js/viewer/viewer.js

```diff
@@ -55,6 +56,9 @@ function Viewer(parent, config) {
         // configure osd
         this.osd = OpenSeadragon(this.config.osd);
 
+        // Added to disable the key tracking temporarily
+        this.osd.innerTracker.keyHandler = null;
```

Comment:
Explain why you're disabling it

Reply:
Added a comment. To disable the key shortcuts to move the image up, down, left, and right. This is to avoid conflict with the text input by the user